### PR TITLE
BC-2313 - Newly created class in the old school year Ticket

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Description

Currently the old school year is automatically selected during class creation. This is due to a wrong sorting in the first place and should be removed.
## Links to Tickets or other pull requests

BC-2313
## Changes

* remove unnecessary sorting before showing available school years in class creation page
## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: controllers/administration.js [hunk 1]
```diff
@@ -1538,8 +1538,7 @@ const renderClassEdit = (req, res, next) => {
 			const mode = req.locals.mode;
 			Promise.all(promises).then(
 				([teachers, gradeLevels, currentClass]) => {
-					let schoolyears = getSelectableYears(res.locals.currentSchoolData);
-					schoolyears = schoolyears.sort((a, b) => a.startDate.localeCompare(b.startDate));
+					const schoolyears = getSelectableYears(res.locals.currentSchoolData);
```
Comment (christian-darsow) [inline controllers/administration.js:1541]: There is probably no easy solution to have the years sorted asc && having the correct year pre-selected on creating a new class, or? This does the trick with correct preselection but the order of the options seems quiet weird, at least to me :)

Comment (MartinSchuhmacher) [reply controllers/administration.js:1541]: exactly, there would be a new logic needed to implement what we actually need. right now the array is filled with the "actual year" in first place and this we use in this solution to get what we need without puting too much effort into old logic

Comment (MartinSchuhmacher) [reply controllers/administration.js:1541]: so the question is: can we live with the order of the years or do we want to implement new logic to have a proper order there

Comment (christian-darsow) [reply controllers/administration.js:1541]: i would then agree, let us live with this order. Correct preselection should probably not make the user open the options anyway

**Review by christian-darsow — APPROVED** [review]

**Pull request merged**